usr/local/www/system_firmware.php

```diff
@@ -102,14 +102,9 @@ function file_upload_error_message($error_code) {
 if(is_subsystem_dirty('firmwarelock')) {
 	$pgtitle = array(gettext("System"),gettext("Firmware"),gettext("Manual Update"));
 	include("head.inc");
-	echo "<body link=\"#0000CC\" vlink=\"#0000CC\" alink=\"#0000CC\">\n";
 	include("fbegin.inc");
-	echo "<div>\n";
-	print_info_box(gettext("An upgrade is currently in progress.<p>The firewall will reboot when the operation is complete.") . "</p><p><img src='/themes/{$g['theme']}/images/icons/icon_fw-update.gif' alt='update' /></p>");
-	echo "</div>\n";
-	include("fend.inc");
-	echo "</body>";
-	echo "</html>";
+	print_info_box(gettext("An upgrade is currently in progress. The firewall will reboot when the operation is complete.") . "<p><img src='/themes/{$g['theme']}/images/icons/icon_fw-update.gif' alt='update' /></p>");
```

Comment:
Yes it is. It is a rather nice animated image so I left it in place. We can always remove it later

> On Apr 30, 2015, at 3:41 PM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/system_firmware.php <https://github.com/SjonHortensius/pfsense/pull/157#discussion_r29462803>:
> 
> >  	include("fbegin.inc");
> > -	echo "<div>\n";
> > -	print_info_box(gettext("An upgrade is currently in progress.<p>The firewall will reboot when the operation is complete.") . "</p><p><img src='/themes/{$g['theme']}/images/icons/icon_fw-update.gif' alt='update' /></p>");
> > -	echo "</div>\n";
> > -	include("fend.inc");
> > -	echo "</body>";
> > -	echo "</html>";
> > +	print_info_box(gettext("An upgrade is currently in progress. The firewall will reboot when the operation is complete.") . "<p><img src='/themes/{$g['theme']}/images/icons/icon_fw-update.gif' alt='update' /></p>");
> Is this image still present in the docroot? I'd guess it's no longer necessary
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/157/files#r29462803>.
> 

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com <mailto:sbeaver@netgate.com> | www.netgate.com <http://www.netgate.com/>